chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -661,6 +661,12 @@ Zotero.Translate.ItemSaver.prototype = {
 	},
 	
 	"_saveNotes":function(item, parentID) {
+		// This can result in a lot of notes being attached if notes is set to string
+		if (typeof item.notes == 'string') {
+			Zotero.debug("WARNING: item.notes set to string. Should be an array");
+			item.notes = [item.notes];
+		}
```

Comment:
I'd rather drop notes/tags in this case, or move it to `Zotero.Translate.Sandbox.Base._itemDone` and make it throw an error.